src/client/Microsoft.Identity.Client/ManagedIdentity/MachineLearningManagedIdentitySource.cs

```diff
@@ -0,0 +1,95 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Globalization;
+using Microsoft.Identity.Client.Core;
+using Microsoft.Identity.Client.Internal;
+
+namespace Microsoft.Identity.Client.ManagedIdentity
+{
+    internal class MachineLearningManagedIdentitySource : AbstractManagedIdentity
+    {
+        // MSI Constants. Docs for MSI are available here https://docs.microsoft.com/azure/app-service/overview-managed-identity
```

Comment:
The link should be for the ML managed identity I think? This one is for app service

Reply:
removed it for now as I am not able to find anything relevant. Checking with ML team.